index.html

```diff
@@ -17,13 +17,14 @@
       <header class="header">
           <h1><a href="#">My Logo</a></h1>
           <nav> 
-            <img src="images/Union.png" alt="hamburgeur menu" class="mobile-img">
-            <!-- <img src="images/desktop-header.svg" alt="hamburgeur menu" class="desktop-img"> -->
+            <a href="#"><img src="images/Union.png" alt="hamburgeur menu" class="mobile-img"></a>
+         
           </nav>
-          <ul class="header-menu">
-            <li>Portoflio</li>
-            <li>About</li>
-            <li>Contact</li>
+          <ul class="header-menu" id="header-menu">
+            <li><a href="#portfolio"  class="link">Portoflio</a></li>
+            <li><a href="#about" class="link">About</a></li>
+            <li><a href="#contact" class="link">Contact</a></li>
+            <li> <a href="#"><img src="images/removenav.svg" alt="Back hamburgeur menu " class="remove mobile-img"></a></li>
```

Comment:
- [ ] Please Follow the Figma design in your mobile menu as the menu items font should be bigger.

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-06-21 17-20-52](https://user-images.githubusercontent.com/33752735/174837675-21859939-31ee-40dd-9e6b-0cd85dfc6788.png) | ![Screenshot from 2022-06-21 16-09-48](https://user-images.githubusercontent.com/33752735/174837701-60414f6e-c88d-4e3c-b020-86bab850f4cd.png)